File: pbwebrcewcl/src/app/appalti/commons/dto/crea-proc-agg-request.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

/* tslint:disable */
import { ProceduraAggiudicazione } from './procedura-aggiudicazione';
export interface CreaProcAggRequest {
  idProgetto?: number;
  proceduraAggiudicazione?: ProceduraAggiudicazione;
}

File: pbwebrce/pbwebrce-jar/src/main/java/it/csi/pbandi/pbwebrce/integration/request/CreaProcAggRequest.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbwebrce.integration.request;

import it.csi.pbandi.pbwebrce.dto.ProceduraAggiudicazione;

public class CreaProcAggRequest {
	private ProceduraAggiudicazione proceduraAggiudicazione;
	private Long idProgetto;
	public ProceduraAggiudicazione getProceduraAggiudicazione() {
		return proceduraAggiudicazione;
	}
	public void setProceduraAggiudicazione(ProceduraAggiudicazione proceduraAggiudicazione) {
		this.proceduraAggiudicazione = proceduraAggiudicazione;
	}
	public Long getIdProgetto() {
		return idProgetto;
	}
	public void setIdProgetto(Long idProgetto) {
		this.idProgetto = idProgetto;
	}
	
	
}
